index.html

```diff
@@ -4,44 +4,69 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  
     <title>Sabka Bazar</title>
-    <!-- <base href="/"> -->
+
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Dosis:400,600,700">
```

Comment:
For now its okay, but this isn't the only way to use web fonts. There are better and different approaches available. Please explore them as well. Read about https://css-tricks.com/fout-foit-foft/